src/components/Video/Video.tsx

```diff
@@ -43,13 +44,24 @@ export const Video = ({
       credit={credit}
       licence={licence}
     >
-      <div className="widescreen-container">
-        <iframe
-          allowFullScreen
-          className="widescreen-media"
-          src={embedSrc}
-          title={caption}
-        />
+      <div className="cc-video">
+        <div className="cc-video__inner">
+          <iframe
+            allowFullScreen
+            className="cc-video__iframe"
+            src={embedSrc}
+            title={caption}
+            aria-describedby="youtube-consent"
+          />
+        </div>
+        <p className="cc-video__caption" id="youtube-consent">
```

Comment:
Will this static id present problems if there is more than one Video on a page?

Reply:
Yes, yes it will. Thanks

Reply:
Updated